packages/terminal/src/widget.ts

```diff
@@ -350,35 +338,17 @@ class Terminal extends Widget {
    * Set the stylesheet.
    */
   private _setStyle(): void {
```

Comment:
Does this need to be its own method now? Why not just set the class in the the `theme` setter?